download-variables.tf

```diff
@@ -2,6 +2,7 @@ variable "dcos_download_path" {
   type = "map"
 
   default = {
+   "1.7.3"      = "https://downloads.dcos.io/dcos/EarlyAccess/commit/14509fe1e7899f439527fb39867194c7a425c771/dcos_generate_config.sh"
```

Comment:
it looks like there is not an official 1.7.3 release for open dcos. You can find all the official releases [here](https://dcos.io/releases/) but I appreciate the suggestion,. I also notices that its the same commit id as 1.7-open.